NuGet.config

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!-- Ignore global configuration -->
+    <clear />
+    <!-- Here so the Visual Studio package manager won't default to the next feed below. -->
```

Comment:
I just remembered that I copied the NuGet file from NEST as a baseline. This comment is [from there](https://github.com/Lombiq/DotNest-Tenants-Core/blob/dev/NuGet.config#L4). Is it also incorrect? (I can't check it since I don't have VS.)